velox/exec/AggregateWindow.cpp

```diff
@@ -341,7 +342,34 @@ class AggregateWindowFunction : public exec::WindowFunction {
     });
 
     // Set null values for empty (non valid) frames in the output block.
-    setNullEmptyFramesResults(validRows, resultOffset, result);
+    setEmptyFramesResult(validRows, resultOffset, emptyResult_, result);
+  }
+
+  // Precompute and save the aggregate output for empty input in emptyResult_.
+  // This value is returned for rows with empty frames.
+  void computeDefaultAggregateValue(
+      const std::string& name,
+      const TypePtr& resultType) {
+    constexpr int kRowSizeOffset = 8;
+    constexpr int kOffset = kRowSizeOffset + 8;
+    aggregate_->setOffsets(kOffset, 0, 1, kRowSizeOffset);
+    std::vector<char> group(kOffset + aggregate_->accumulatorFixedWidthSize());
+    std::vector<char*> groups(1, group.data());
+    aggregate_->initializeNewGroups(
+        groups.data(), std::vector<vector_size_t>{0});
+
+    // addSingleGroupRawInput is called to catch the expected errors for
```

Comment:
Do you have an example of a fuzzer error ? We can give an example here back-tracing from that.